### ISSUE TITLE
[Accessibility][Accordion][CSS] chevron_right and expand_more icons are read by NVDA screen reader.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1783]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Use TAB and TAB + SHIFT to navigate to an accordion trigger.

**Current Behavior**
Text alternatives for decorative icons in buttons are provided through the CSS content property, which screen reader users may or may not hear announced. In this case, the `content` property is being read aloud and NVDA announces text that is not meaningful, e.g. "chevron_right".

**Expected behavior/code**
If an icon is purely decorative, it should be hidden from assistive technology.

**Environment**
Chrome 92, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.1.1 Non-Text Content

**Possible Solution**
Consider using an empty CSS "content" property value; alternatively, add icons with an html element that can be hidden from assistive technology with `aria-hidden="true"`; use a CSS background image; or use an `img` element with a null alt attribute.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
Please make sure that this also fixes #1672.
